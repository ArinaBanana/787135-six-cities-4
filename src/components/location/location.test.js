import React from "react";
import renderer from "react-test-renderer";
import Location from "./location.jsx";

it(`Should render Location`, () => {
  const tree = renderer.create(<Location city={`City`} isActive={false} onTitleClick={() => {}} />).toJSON();

  expect(tree).toMatchSnapshot();
});
