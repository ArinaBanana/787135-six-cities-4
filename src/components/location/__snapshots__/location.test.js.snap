// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Location 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item"
    href="#"
  >
    <span>
      City
    </span>
  </a>
</li>
`;
